aggregated_points_goals.py

```diff
@@ -0,0 +1,30 @@
+import pandas as pd
+import numpy as np
+
+def modifyTable(data):
+    team_matches = data.drop_duplicates(subset=['season_x','team_x','GW']).copy()
+    
+    team_matches.loc['points'] = 0
+    for index,row in team_matches.iterrows():
+        isHome = row['was_home']
+        if row['team_h_score'] > row['team_a_score']:
+            team_matches.loc[index,'points'] = 3 - (0 if isHome else 3)
+        elif row['team_a_score'] > row['team_h_score']:     
+            team_matches.loc[index,'points'] = 0 + (0 if isHome else 3)
+        else:
+            team_matches.loc[index,'points'] = 1  
+    team_matches.loc[:,'points'] = team_matches.groupby(['team_x','season_x'])['points'].cumsum()    
+
+    team_matches['team_goals_scored'] = 0
+    team_matches['team_goals_conceded'] = 0
+
+    for index,row in team_matches.iterrows():
+        team_matches.loc[index,'team_goals_scored'] = team_matches.loc[index,'team_h_score'] if row['was_home'] else team_matches.loc[index,'team_a_score']
+        team_matches.loc[index,'team_goals_conceded'] = team_matches.loc[index,'team_a_score'] if row['was_home'] else team_matches.loc[index,'team_h_score']
+
+    team_matches['team_goals_scored'] = team_matches.groupby(['team_x','season_x'])['team_goals_scored'].cumsum()
+    team_matches['team_goals_conceded'] = team_matches.groupby(['team_x','season_x'])['team_goals_conceded'].cumsum()
+    team_matches['team_goals_diff'] = team_matches['team_goals_scored'] - team_matches['team_goals_conceded']
+    newData = data.merge(team_matches[['season_x','GW','team_x','points','team_goals_scored','team_goals_conceded','team_goals_diff']], on=['season_x','GW','team_x'],how='left')
+    return newData
+    
```

Comment:
Could you also please include a main function that reads the input data from the required csv file, executes modifyTable, and then creates the new csv file.

Reply:
Added main and re-created the file under "cleaned_merged_seasons_team_aggregated.csv"